src/integration/InterfaceUtils.cpp

```diff
@@ -19,6 +19,19 @@
 
 using namespace knowrob;
 
+// enum for epistemicOperator
+enum EpistemicOperator {
```

Comment:
It might be confusing that we also have the enum `ModalType` in `ModalOperator.h`.
Can't we unify these?

Reply:
As discussed: It would be nicer for the ROS messages (and also future REST definitions etc.) to have each modaltype start counting with 0